docs/src/app/components/pages/components/SVGIcon/ExampleHover.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
> When I take it off, it gives a lint error of needing react in the scope when using JSX.

Yes, that's because `<MyComponent` is transformed into `React.creatComponent`.
I'm talking about `docs/src/app/components/pages/components/SVGIcon/ExampleHover.jsx`